micromamba/src/clean.cpp

```diff
@@ -70,6 +76,16 @@ set_clean_command(CLI::App* subcom)
             options = options | MAMBA_CLEAN_LOCKS;
         if (config.at("clean_trash").compute().value<bool>())
             options = options | MAMBA_CLEAN_TRASH;
+        if (config.at("clean_force_pkgs_dirs").compute().value<bool>())
+        {
+            std::cout << "Remove all contents from the package caches ([y]/n)?" << std::endl;
```

Comment:
Can you use `Console::prompt(...)`?

Reply:
And mabye we should move thsi to the actual place where things are removed? And print what package dirs are going to be erased.